infra/main.tf

```diff
@@ -102,3 +102,58 @@ resource "aws_cloudwatch_event_target" "cloudwatch_event_target" {
     "table" : aws_dynamodb_table.dynamodb_table.name
   })
 }
+
+resource "aws_elasticsearch_domain" "elasticsearch_domain" {
+  domain_name           = "tf-creds-es-domain"
```

Comment:
Do we maybe want to add this name to variables.tf ?
We have sticked to this method in the poc - I think it can be good to stick to it here as well

Reply:
yup. Good idea. Will implement it in a seperate PR unless you want to take a shot at it.